core/sbt-cloudflow/src/main/scala/cloudflow/sbt/BlueprintVerificationPlugin.scala

```diff
@@ -66,7 +66,7 @@ object BlueprintVerificationPlugin extends AutoPlugin {
     },
     applicationDescriptor := {
       val appId           = (ThisProject / name).value
-      val agentPathsMap   = agentPaths.value
+      val agentPathsMap   = Map("prometheus" -> "/dummy")
```

Comment:
is this for backwards compatibility?

Reply:
yes, But I changed it just now to remove the mapping